static/sass/styles.scss

```diff
@@ -638,7 +631,31 @@ $p-small-lh-diff: map-get($line-heights, default-text) - map-get($line-heights,
 
     @media screen and (max-width: 688px) {
       height: 181px;
-      width: 121px;  
+      width: 121px;
+    }
+  }
+}
+
+.p-overlayed-section {
+  display: grid;
+  grid-template-columns: 2fr 8fr;
+  grid-template-rows: 477px 106px 1fr;
```

Comment:
This feels very fragile, and won't update when the value of 1rem changes after 1680px. Can this be done with just a regular strip wrapping this and then using the variable in the regular strip for an overlap value?